### PR TITLE
adding info about code archive

### PR DESCRIPTION
**Pull request by nataliejeremy**
This closes issue #426. @suzannealdrich Please review--thanks!

### CODE REVIEW

**Comment by suzannealdrich** [comment]
:+1:

File: source/docs/articles/sites/backups/cloning-an-existing-site-from-a-dashboard-backup.md [hunk 1]
```diff
@@ -13,9 +13,11 @@ All Pantheon sites consist of three parts:
 
 ## Copy Your Code/Files/Database
 
-1. From your Site Dashboard, go to the live environment and click **Backups**.
+1. From your Site Dashboard, go to the Live environment and click **Backups**.
 2. Select the backup you want to clone from, and download each of the backup files (Code, Database, Files) by clicking the **download** icon.
 
+When a code archive is downloaded for an environment, it contains the full Git repository checked out at master. If this code archive is combined with the database and files archive from that point in time, the code might not match the snapshot deployed on that environment. If you want to make a clone of Live or Test and it is a few commits behind Dev,  use Git to checkout the appropriate tag for the state of code you want. This will leave the repository in a detached head state so you can create and then merge a branch back into master prior to uploading it.
```
Comment (bmackinney) [inline source/docs/articles/sites/backups/cloning-an-existing-site-from-a-dashboard-backup.md]: Consider adding the command `git tag` to list tags, and `git checkout master <tag>` i.e. `git checkout master pantheon_live_5` for backups from live where live_5 represents the highest *pantheon_live_n* tag, and there are commits more recent than the last deploy to Live.

**Comment by nataliejeremy** [comment]
@bmackinney Can you incorporate that?

**Comment by nataliejeremy** [comment]
@bmackinney please review the new content I added.

**Comment by nataliejeremy** [comment]
@bmackinney Please review and merge.

File: source/docs/articles/sites/backups/cloning-an-existing-site-from-a-dashboard-backup.md [hunk 2]
```diff
@@ -13,9 +13,14 @@ All Pantheon sites consist of three parts:
...
 ## Copy Your Code/Files/Database
 
-1. From your Site Dashboard, go to the live environment and click **Backups**.
+1. From your Site Dashboard, go to the Live environment and click **Backups**.
 2. Select the backup you want to clone from, and download each of the backup files (Code, Database, Files) by clicking the **download** icon.
 
+When a code archive is downloaded for an environment, it contains the full Git repository checked out at master. If this code archive is combined with the database and files archive from that point in time, the code might not match the snapshot deployed on that environment. If you want to make a clone of Live or Test and it is a few commits behind Dev,  use Git to checkout the appropriate tag for the state of code you want. This will leave the repository in a detached head state so you can create and then merge a branch back into master prior to uploading it.  
+**Example**  
+`git tag`  
+`git checkout master <tag>` i.e. git checkout master pantheon_live_5 for backups from live where live_5 represents the highest pantheon_live_n tag, and there are commits more recent than the last deploy to Live.
```
Comment (bmackinney) [inline source/docs/articles/sites/backups/cloning-an-existing-site-from-a-dashboard-backup.md]: looks like we need a few more commands. @suzannealdrich after `git checkout master <tag>`, will `git checkout -b branch`, `git checkout master`, and `git merge branch` work or can users `git rebase --onto pantheon_live_n master`?

Comment (suzannealdrich) [inline source/docs/articles/sites/backups/cloning-an-existing-site-from-a-dashboard-backup.md]: To get the master branch in the same state as the code checked out on a tag:
```
git checkout pantheon_env_nnn 
git checkout -b new_branch_name
git checkout master
git merge new_branch_name
```
Then you can make an archive of this codebase to import into a new site, or you can push the master branch back up to the origin, or push the new branch and try it out on a multidev.

**Pull request merged**